src/game/option.c

```diff
@@ -47,16 +47,24 @@ typedef enum COMPASS_TEXT {
 } COMPASS_TEXT;
 
 typedef enum DETAIL_HW_TEXT {
-    DETAIL_HW_TITLE = 0,
-    DETAIL_HW_TITLE_BORDER = 1,
-    DETAIL_HW_PERSPECTIVE = 2,
-    DETAIL_HW_BILINEAR = 3,
-    DETAIL_HW_UI_TEXT_SCALE = 4,
-    DETAIL_HW_UI_BAR_SCALE = 5,
+    DETAIL_HW_PERSPECTIVE = 0,
+    DETAIL_HW_BILINEAR = 1,
+    DETAIL_HW_UI_TEXT_SCALE = 2,
+    DETAIL_HW_UI_BAR_SCALE = 3,
+    DETAIL_HW_TITLE = 4,
+    DETAIL_HW_TITLE_BORDER = 5,
     DETAIL_HW_RESOLUTION = 6,
     DETAIL_HW_NUMBER_OF = 7,
```

Comment:
Pressing down on `UI bar scale` takes you to the title (Select Detail), pressing down again makes the border disappear and down again gets you to the resolution.

Reply:
Fixed + allowed changing the resolution from within the game, but it requires closing the inventory to apply.